app/src/main/res/layout/fragment_ai_prompt.xml

```diff
@@ -0,0 +1,110 @@
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  ~ GNU General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU General Public License
+  ~ along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  -->
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".ui.main.newMessage.AiPromptFragment">
+
+    <com.google.android.material.card.MaterialCardView
+        style="@style/AiCardView"
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
+        android:clickable="true"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintWidth_max="@dimen/material_bottom_sheet_max_width"
+        app:rippleColor="@android:color/transparent"
+        tools:ignore="KeyboardInaccessibleWidget">
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical"
+            android:padding="@dimen/marginStandardMedium">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_marginBottom="@dimen/marginStandardMedium"
+                android:gravity="center_vertical"
+                android:orientation="horizontal">
+
+                <ImageView
+                    android:layout_width="@dimen/mediumIconSize"
+                    android:layout_height="@dimen/mediumIconSize"
+                    android:layout_marginEnd="@dimen/marginStandardSmall"
+                    android:importantForAccessibility="no"
+                    android:src="@drawable/ic_ai_magic_wand" />
+
+                <TextView
+                    android:id="@+id/aiPromptTitle"
+                    style="@style/H2"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    tools:text="@string/aiPromptTitle" />
+
+                <Space
+                    android:layout_width="0dp"
+                    android:layout_height="0dp"
+                    android:layout_weight="1" />
+
+                <com.google.android.material.button.MaterialButton
+                    android:id="@+id/closeButton"
+                    style="@style/IconButtonSmall"
+                    android:theme="@style/AiCursorAndPrimaryColorTheme"
+                    app:icon="@drawable/ic_close_small" />
```

Comment:
The button's icon is not aligned with the other buttons of the app (24dp insteand of 16)